### DISCUSSION TITLE
Help！when i connected to notion in my locoal dify ，it always had a "invalid_request " error

### DISCUSSION THREAD

**Discussion by candyKou** [langgenius/dify#9395] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

when i connected notion in my locoal dify which viersion is 0.9.2 ，**it always had a "invalid_request " error**.
`{
  "error": "invalid_request",
  "error_description": "query failed validation: query.client_id should be a string or `undefined`, instead was `0`.",
  "request_id": "a4d3f52d-6146-4de9-bf92-2bda1c898a7e"
}`

I have tried all the methods related to this error.But still no solution.

If there will be a solution, I would be very grateful.

**Comment by lachinhan** [comment]
the same .
